test/fn.test.satyg

```diff
@@ -0,0 +1,17 @@
+@require: test/test
+@require: test/expect
+
+@import: ../src/fn
+
+let fn-test-cases = open Test in
+  describe `Fn module` [
+    it `make recursive function with fix` (fun x -> (
```

Comment:
The original test case calls `Debug.log` for each recursion, but it is not feasible in satysfi-test. That's why I write this case in this way.